src/components/forms/AddItemForm.tsx

```diff
@@ -63,23 +76,30 @@ export function AddItemForm({ listPath, data: unfilteredListItems }: Props) {
 			return;
 		}
 
+		if (itemQuantity < 1) {
+			toast.error("Oops! Item quantity must be more than 0!");
+			return;
+		}
```

Comment:
Found a bug:
After submitting one item with a specific quantity, staying on the same page to submit a new item I am unable to edit the quantity to a different number. I also see the `0` quantity get frozen there. See my replication below:
![Sep-28-2024 10-43-05](https://github.com/user-attachments/assets/80df8c8e-2246-44a4-b2f0-65044d8220be)

Reply:
Thank you for noticing! And testing!
All of this should be resolved now that I set the item quantity as it's own label and input in the Add Item form. The number input is working properly, which is answering ALL of my questions, and fixing most of the bugs/stuff I was worried about.

Plus, when the form is submitted, all the inputs are reset.

...........I thought my replies and screenshots were going to this.